client/src/components/Rooms/RoomCard/RoomCard.js

```diff
@@ -32,6 +33,10 @@ const roomCard = props => {
             <p>{props.description}</p>
             <Grid container justify="space-evenly" className="back-btn-wrapper">
               <span>Members: {props.members} / {props.limit}</span>
```

Comment:
Should this line be deleted?

Reply:
I will fix "Sign Up" bug in another pull request (when I will connect redux store to the SignUp component). This happens because Sign Up component only update App state (not redux store), but AppHeader watching only for redux store...
RoomCard resolved...thanks:)